### PR TITLE
Adds proper logging to drop-bomb admin verb.

### PR DESCRIPTION
**Pull request by lzimann**
For some reason it didn't log to the file at all, it just messaged admins which does not log.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
w-w-why would you want to do this?

**Comment by kevinz000** [comment]
feature?

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -521,7 +522,8 @@ var/list/admin_verbs_hideable = list(
 				return
 			epicenter = mob.loc //We need to reupdate as they may have moved again
 			explosion(epicenter, devastation_range, heavy_impact_range, light_impact_range, flash_range)
-	message_admins("<span class='adminnotice'>[ckey] creating an admin explosion at [epicenter.loc].</span>")
+	message_admins("<span class='adminnotice'>[ADMIN_LOOKUPFLW(usr)] creating an admin explosion at [epicenter.loc].</span>")
```
Comment (Jordie0608) [inline code/modules/admin/admin_verbs.dm]: No span class needed for `message_admins`

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -533,7 +535,8 @@ var/list/admin_verbs_hideable = list(
 	var/turf/epicenter = mob.loc
 	if(ex_power && epicenter)
 		dyn_explosion(epicenter, ex_power)
-		message_admins("<span class='adminnotice'>[ckey] creating an admin explosion at [epicenter.loc].</span>")
+		message_admins("<span class='adminnotice'>[ADMIN_LOOKUPFLW(usr)] creating an admin explosion at [epicenter.loc].</span>")
```
Comment (Jordie0608) [inline code/modules/admin/admin_verbs.dm]: Same

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

**Review by Jordie0608 — APPROVED** [review]

**Pull request merged**